AbletonMCP_Remote_Script/handlers/browser.py

```
@@ -0,0 +1,641 @@
+                        count += 1
+                        search_item(child, depth + 1, max_depth)
+
+        if category == "all" or category not in _BROWSER_ROOTS:
+            for attr in _BROWSER_ROOTS:
+                if attr == "user_folders":
+                    # user_folders is a list of BrowserItems, not a single root
+                    try:
+                        for folder in getattr(app.browser, "user_folders", []):
+                            search_item(folder)
+                    except Exception:
+                        pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)